tests/bluetooth/test_descriptor_emulation.py

```diff
@@ -265,3 +285,135 @@ async def test_bluetooth_add_descriptor_to_unknown_characteristic(
             websocket, context_id, device_address, HEART_RATE_SERVICE_UUID,
             MEASUREMENT_INTERVAL_CHARACTERISTIC_UUID,
             CHARACTERISTIC_USER_DESCRIPTION_DESCRIPTOR_UUID, 'add')
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('capabilities', [{
+    'goog:chromeOptions': {
+        'args': ['--enable-features=WebBluetooth']
+    }
+}],
+                         indirect=True)
+async def test_bluetooth_descriptor_write_event(websocket, context_id, html):
+    await setup_dscriptor(websocket, context_id, html, HEART_RATE_SERVICE_UUID,
+                          MEASUREMENT_INTERVAL_CHARACTERISTIC_UUID,
+                          {'write': True},
+                          CHARACTERISTIC_USER_DESCRIPTION_DESCRIPTOR_UUID)
+    await subscribe(websocket, [DESCRIPTOR_EVENT_GENERATED])
+    expected_data = [1]
```

Comment:
Can we make the test case a bit less trivial?
```suggestion
    expected_data = [42, 27]
```